pages/api/auth/[...nextauth].ts

```diff
@@ -156,6 +159,7 @@ export const authOptions: NextAuthOptions = {
         ...(token.newlyRegistered && { newlyRegistered: token.newlyRegistered }),
         // Used client side to immidiately log out a user if they have been deactivated
         ...(token.deactivated && { deactivated: token.deactivated }),
+        securityQuestions: await retrieveUserSecurityQuestions({ userId: token.userId as string }),
```

Comment:
Is it necessary to have the security questions added to the session?  This adds a DB call everytime the client side useSession refreshes or on page navigation.

Reply:
Before we make any decisions I'd like to continue to go through the rest of the code.  Still getting a feeling for how everything works together.

Reply:
Resolved via https://github.com/cds-snc/platform-forms-client/pull/2497